README.md

```diff
@@ -1,2 +1,155 @@
-# bedrock-web-data-hub
-Web-based Bedrock Data Hub
+# Web Based Bedrock Data Hub Adapter _(bedrock-web-data-hub)_
+
+[![Build Status](https://travis-ci.org/digitalbazaar/bedrock-web-data-hub.png?branch=master)](https://travis-ci.org/digitalbazaar/bedrock-web-data-hub)
+
+> A Javascript library for Bedrock web apps, for interfacing with a remote Data Hub server
+
+## Table of Contents
+
+- [Background](#background)
+- [Install](#install)
+- [Usage](#usage)
+- [API](#api)
+- [Contribute](#contribute)
+- [Commercial Support](#commercial-support)
+- [License](#license)
+
+## Background
+
+This library is an adapter that Bedrock web apps can use to interface with
+remote Data Hub servers (for example, it's used by the 
+[Bedrock VC Store](https://github.com/digitalbazaar/bedrock-web-vc-store) lib).
+
+It consists of two main classes:
+
+1. `DataHubService` - stores and manages `DataHub` configs with a remote 
+  [Data Hub storage server](https://github.com/digitalbazaar/bedrock-data-hub-storage).
+  Used to create or fetch `DataHub` instances.
+2. `DataHub` - provides a CRUD (+ find) interface to a specific Data Hub. Also
+  ensures appropriate db indexes are set up, and provides some key management
+  under the hood.
+
+## Install
+
+To install locally (for development):
+
+```
+git clone https://github.com/digitalbazaar/bedrock-web-data-hub.git
+cd bedrock-web-data-hub
+npm install
+```
+
+(Optional) Also install test dependencies:
+
+```
+cd test
+npm install
+cd ..
+```
+
+## Usage
+
+### Creating and registering a DataHub config
+
+Setting up a Data Hub Service requires configuring a Key Management Service
+([`bedrock-web-kms`](https://github.com/digitalbazaar/bedrock-web-kms)) instance.
```

Comment:
Just to be clear, this isn't quite accurate. The KMS stuff really has to do with creating a Data Hub configuration. Furthermore, it's actually not *required* to put a default kek and hmac into a Data Hub config, but we do that now because it's so far clearly the most convenient way to use a Data Hub.

Suffice it to say that you can use Data Hubs (including the bedrock-web-data-hub implementation) by doing all of your own key management on your own. So perhaps we want to just recommend using bedrock-web-kms to handle KMS stuff and provide that in the examples.

Reply:
Understood, I'll adjust the wording. (I'm ok with not getting into documenting how to actually do your own key mgmt for now)